server.js

```diff
@@ -1,12 +1,19 @@
 require("dotenv").config();
+
 const express = require("express");
+const cors = require("cors");
 const voucherifyClient = require("voucherify");
 const app = express();
 const bodyParser = require("body-parser");
 const session = require("express-session");
 const SQLiteStore = require("connect-sqlite3")(session);
 
-const debug = false;
+if (process.env.NODE_ENV !== 'production') {
+  app.use(cors({
+    credentials: true,
+    origin: "http://localhost:3001", // REACT_APP_API_URL
+  }));
+}
```

Comment:
you need to have proper CORS configuration in your server, in case your website (i.e. static files) which is hosted at server A, wants to talk with a an API which is hosted on a server B.

For production, we've only got one server - the express server, which serves all the static files from the `./build` folder